includes/emails/email-summary/class-edd-email-summary.php

```diff
@@ -0,0 +1,791 @@
+
+							<div class="pro-tip-holder" style="background: #F3F8FE; border-radius: 10px; padding: 32px 40px;">
+
+								<div class="pro-tip-section-title" style="font-weight: 600; font-size: 18px; line-height: 23px; color: #2794DA;">
+									<img src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/icons/icon-megaphone.png' ); ?>" alt="#" title="#" width="24" height="24" style="vertical-align: bottom; display: inline-block; margin-right: 4px;">
+									<?php echo esc_html( __( 'Pro-tip from our expert', 'easy-digital-downloads' ) ); ?>
+								</div>
+
+								<div class="pro-tip-title pull-down-12" style="margin-top: 12px; font-weight: 600; font-size: 20px; line-height: 26px; color: #1F2937;">
+									<?php echo esc_html( $blurb['headline'] ); ?>
+								</div>
+
+								<p class="bigger pull-down-8" style="margin: 0px; font-weight: 400; color: #4B5563; margin-top: 8px; font-size: 16px; line-height: 22px;">
+									<?php echo esc_html( $blurb['content'] ); ?>
+								</p>
+
+								<div class="pull-down-15" style="margin-top: 15px;">
+									<a href="<?php echo esc_attr( $this->add_blurb_tracking_parameters( $blurb['button_link'], $blurb ) ); ?>" class="cta-btn" style="padding: 6px 15px; font-weight: 600; font-size: 14px; line-height: 20px; background: #2794DA; display: inline-block; text-decoration: none; color: white;">
+										<?php echo esc_html( $blurb['button_text'] ); ?>
+									</a>
+								</div>
+
+							</div>
+
+						</div>
+
+					</div>
+
+				</div>
+				<!-- /.pro-tip-blurb -->
+				<?php endif; ?>
+
+			</body>
+		</html>
+		<?php
+		$email_template = ob_get_clean();
+		return $email_template;
+	}
+
+	/**
+	 * Add tracking parameters to the blurb URL.
+	 *
+	 * @since 3.1
+	 *
+	 * @param  string $url Destination URL.
+	 * @param  array $blurb Blurb data.
+	 * @return string URL with tracking parameters.
+	 */
+	public function add_blurb_tracking_parameters( $url, $blurb ) {
+		// @todo - Define tracking parameters!
+		return add_query_arg(
+			array(
+				'utm_source'   => 'email-summary',
+				'utm_medium'   => 'blurb-' . rawurlencode( $blurb['headline'] ), // @todo - We have to change this to something more appropriate.
+				'utm_campaign' => 'email',
+				'utm_content'  => 'pro-tip',
+			),
+			$url
+		);
+	}
+
+	/**
+	 * Prepare and send email.
+	 *
+	 * @since 3.1
+	 *
+	 * @return bool True if email was sent, false if there was an error.
+	 */
+	public function send_email() {
+		// Get next blurb.
+		$email_blurbs = new EDD_Email_Summary_Blurb();
+		$next_blurb   = $email_blurbs->get_next();
+
+		// Prepare email.
+		$email_subject    = $this->get_email_subject();
+		$email_recipients = $this->get_email_recipients();
+		$email_body       = $this->build_email_template( $next_blurb );
+		$email_headers    = array( 'Content-Type: text/html; charset=UTF-8' );
+
+		// If there is no email body, do not send an email.
+		if ( ! $email_body ) {
```

Comment:
Maybe getting the email body and doing this early return could be moved to the top of the method so that nothing else would run if it's empty.